src/com/itacademy/eshop/product/Product.java

```diff
@@ -33,6 +37,17 @@ public ArrayList<Review> getReviews() {
         return reviews;
     }
 
+    public void setDescription(String productDescription) {
```

Comment:
when setting a value you should pass the Description object instead of creating one inside the setter method from the needed values